doc/modules/classifier_chain.rst

```diff
@@ -0,0 +1,32 @@
+.. _classifier_chain:
+
+================
+Classifier Chain
+================
+
+.. currentmodule:: sklearn.classifier_chain
```

Comment:
This module does not exist. The appropriate place for documentation appears to be `doc/modules/multiclass.rst`